### DISCUSSION TITLE
[@types/jquery] add .length method

### DISCUSSION THREAD

**Discussion by MahdiAkrami01** [DefinitelyTyped/DefinitelyTyped#55576] [Issues with a @types package]
please add this method :
https://api.jquery.com/length/
i add jquery in phpstorm js lib but this function not suggested.

```
var a = $('.some-div')

console.log(a.length)
```

[a comment by an automated account is not shown]

**Comment by leonard-thieu** [comment]
This already exists.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4d9b1134a6d50a3fcdd81d23f95d836b9c10d8a1/types/jquery/JQuery.d.ts#L29-L76
